### ISSUE TITLE
Does this work on the MacBook Pro 16" 2019?

### ISSUE THREAD

**Issue by jadenquinn** [0xbb/gpu-switch#60]

**Comment by atrinb3** [comment]
Nope. Windows won't load when you switch to the iGPU, even when it's enabled in device manager.

**Comment by wuxiaohong11** [comment]
I am also a 2019 model. When I started using efi to introduce the system, I could see dual Gpus, but once I restarted, I couldn't see IGpus
